chainstate/client_options.go

```diff
@@ -241,3 +241,10 @@ func WithBroadcastClientAPIs(apis []broadcastClient.ArcClientConfig) ClientOps {
 		c.config.broadcastClientConfig.BroadcastClientApis = apis
 	}
 }
+
+// WithPulse will set pulse client APIs
+func WithPulse(url, authToken string) ClientOps {
+	return func(c *clientOptions) {
+		c.config.pulseClient = &PulseClient{url, authToken}
```

Comment:
To make it "client" friendly we should make some validation as fast as possible in a code, that if someone choose to use BEEF and not set up Pulse, then we should fail. Actually the best would be to shutdown bux if someone does such a missconfiguration.

And now, I'm thinking that maybe, actually configuration for pulse should be required by the function WithBEEF (of how it is called). WDYT?

Reply:
Done